src/test/java/com/training/baseTest/WebBaseTest.java

```diff
@@ -0,0 +1,28 @@
+package com.training.baseTest;
```

Comment:
As told earlier, package name should not be in camel case. It should be all in lower case
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html